cfme/containers/container.py

```diff
@@ -44,8 +45,18 @@ def navigate(self, detail=True):
                     'containers_container_detail', context={
                         'container': self, 'pod': self.pod})
         else:
-            sel.force_navigate('containers_container', context={'container': self})
+            sel.force_navigate('containers_container',
+                               context={'container': self, 'pod': self.pod})
```

Comment:
Think about the nav tree logic. If container provides a pod object, then it is pointless to pass the pod along.

The nav tree pulls the pod from container, line 16 and 20 ;)